test/Target/ExportSystemC/basic.mlir

```diff
@@ -0,0 +1,5 @@
+// RUN: circt-translate %s --export-systemc | FileCheck %s
+
+// CHECK-LABEL: SC_MODULE(basic) {
+// CHECK-NEXT: };
+systemc.module @basic () { }
```

Comment:
There is a fair amount more in the emitter that isn't being tested here. E.g., `ifndef` stuff. That should be included.